crates/exex/types/src/lib.rs

```diff
@@ -33,3 +33,6 @@ impl FinishedExExHeight {
         matches!(self, Self::NotReady)
     }
 }
+
+/// The thresholds at which the backfill stops executing the range of blocks and returns the result.
+pub type BackfillThresholds = reth_stages_types::ExecutionStageThresholds;
```

Comment:
please don't add aliases for these

Reply:
that was a nice try to get an alias in, ok boss